public/js/components/Info.js

```diff
@@ -0,0 +1,148 @@
+const React = require("react");
+const { connect } = require("react-redux");
+const classnames = require("classnames");
+const { getTabs } = require("../selectors");
+
+require("./Info.css");
+const { DOM: dom } = React;
+const ImPropTypes = require("react-immutable-proptypes");
+
+const githubUrl = "https://github.com/devtools-html/debugger.html/blob/master";
+
+function getTabsByBrowser(tabs, browser) {
+  return tabs.valueSeq()
+    .filter(tab => tab.get("browser") == browser);
+}
+
+function firstTimeMessage(title, urlPart) {
+  return dom.div(
+    { className: "footer-note" },
+    `First time connecting to ${title}? Checkout out the `,
+    dom.a({ href: `${githubUrl}/CONTRIBUTING.md#${urlPart}` }, "docs"),
+    "."
+  );
+}
+
+const Info = React.createClass({
```

Comment:
Hmm, that's a good point. I believe `LandingPage` is more descriptive, while not too limiting.